vector/src/main/java/im/vector/app/features/home/room/detail/timeline/item/IsEventItem.kt

```diff
@@ -14,12 +14,12 @@
  * limitations under the License.
  */
 
-package im.vector.app.core.epoxy
+package im.vector.app.features.home.room.detail.timeline.item
 
-import com.airbnb.epoxy.EpoxyModelClass
-import im.vector.app.R
-
-@EpoxyModelClass(layout = R.layout.item_empty)
-abstract class EmptyItem : VectorEpoxyModel<EmptyItem.Holder>() {
-    class Holder : VectorEpoxyHolder()
+interface IsEventItem {
```

Comment:
What do you mean by `IsEventItem`?

Reply:
It's an item which holds one or several item ids. If you have a better name suggestion, because I wasn't inspired...

Reply:
I propose `ItemWithEvents`